src/Log.hpp

```diff
@@ -38,8 +41,11 @@ class LoggerStream : public std::ostringstream
 
 extern Logger g_nLog;
 
+#define NOTIFIER_LOGGER usbguardNotifier::g_nLog
+
 #define NOTIFIER_LOG() \
```

Comment:
make this a function maybe

Reply:
What do you mean by this?

Reply:
NOTIFIER_LOG() ... why is this not a function

Reply:
Next time please mark the correct line. It's better to let``NOTIFIER_LOG``  stay like this for the following reason: we do want to know from which file, function and line did we inkove the logger. If you add this part to a function, you will get the same results whenever you call it.

Reply:
oh ofc I forgot about that